Add April front end release (#426)
SenseNet/sensenet.github.io: _updates/2020-04-08-april.md

---
title: April 2020
---

## Work From Home 🏠

> Because of the pandemic we are now working from home. This doesn't mean that the developement stopped. Here are the changes we made.

Reviewer: ```suggestion
Because of the pandemic we are now working from home. This doesn't mean that the development is stopped. Here are the changes we made.
```

### @sensenet/redux@5.3.0

- Added some missing actions for
  - put

>   - content creation with content template

Reviewer: ```suggestion
  - content creation with a content template
```

  - preview related actions
  - preview comments
  - allowedchildtypes
  - restore from trash
  - add/remove list fields

### @sensenet/document-viewer-react@2.0.6

> - fix: adding comments returned error always, because the x and y coordinates were floating numbers and the request body was not a valid json. These two params' type is changed to string. [#617](https://github.com/SenseNet/sn-client/pull/617)

Reviewer: ```suggestion
- fix: adding comments returned error, because the x and y coordinates were floating numbers and the request body was not a valid json. These two params' type is changed to string. [#617](https://github.com/SenseNet/sn-client/pull/617)
```

### sn-dms-demo@2.0.1

> - Refactored dms demo to use create react app.

Reviewer: ```suggestion
- DMS demo is refactored to use create react app.
```
